.github/workflows/test.yml

```diff
@@ -200,7 +200,7 @@ jobs:
       - name: Setup GPG (for Publish)
         uses: olafurpg/setup-gpg@v3
       - name: Publish
-        run: sbt ci-release
+        run: sbt -DdisableFatalWarnings ci-release
```

Comment:
Can you make sure we're building the docs in CI before this publish step? Something like adding `sbt -DdisableFatalWarnings doc` up above, above the `Build the microsite` step. And maybe add a comment like `# TODO make the microsite building include building ScalaDoc`.